x-pack/legacy/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -41,7 +48,7 @@ export async function setupRequest(req: Legacy.Request) {
     start: moment.utc(query.start).valueOf(),
     end: moment.utc(query.end).valueOf(),
     uiFiltersES,
-    client: getESClient(req),
+    client: getESClient(req, { asInternalUser: clientAsInternalUser }),
```

Comment:
I think you should just leave the name as `clientAsInternalUser`. I find having multiple names for the same thing is often (not always) more confusing that it does good

Reply:
... and I think `asInternalUser` everywhere is better than `clientAsInternalUser` everywhere